src/components/Checkbox.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { StyleSheet, Text, Pressable, View } from 'react-native'
```

Comment:
Recuerdan que al usar texto siempre usaremos el componente `StyledText` en vez de `Text`

```suggestion
import { StyleSheet, StyledText, Pressable, View } from 'react-native'
```